galaxy_ng/app/settings.py

```diff
@@ -28,13 +33,15 @@
 GALAXY_PAGINATION_CLASS = "galaxy_ng.app.api.pagination.LimitOffsetPagination"
 GALAXY_AUTHENTICATION_CLASSES = [
     "rest_framework.authentication.SessionAuthentication",
-    "rest_framework.authentication.BasicAuthentication",
-    # "galaxy_ng.app.auth.auth.RHIdentityAuthentication",
+    "rest_framework.authentication.TokenAuthentication",
 ]
 GALAXY_PERMISSION_CLASSES = [
     'rest_framework.permissions.IsAuthenticated',
-    # 'galaxy_ng.app.auth.auth.RHEntitlementRequired',
 ]
+# Settings for insights mode
+# GALAXY_AUTHENTICATION_CLASSES = ["galaxy_ng.app.auth.auth.RHIdentityAuthentication"]
```

Comment:
dynaconf does provide some capabilities to have different settings for different environments. Just an FYI https://dynaconf.readthedocs.io/en/latest/guides/usage.html#settings-file-formats